[2023-02-02T03:10:49.349+0530] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: NSE_Open_Interest.extract scheduled__2023-01-22T00:00:00+00:00 [queued]>
[2023-02-02T03:10:49.373+0530] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: NSE_Open_Interest.extract scheduled__2023-01-22T00:00:00+00:00 [queued]>
[2023-02-02T03:10:49.374+0530] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-02-02T03:10:49.374+0530] {taskinstance.py:1284} INFO - Starting attempt 3 of 4
[2023-02-02T03:10:49.374+0530] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-02-02T03:10:49.403+0530] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): extract> on 2023-01-22 00:00:00+00:00
[2023-02-02T03:10:49.410+0530] {standard_task_runner.py:55} INFO - Started process 4827 to run task
[2023-02-02T03:10:49.414+0530] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'NSE_Open_Interest', 'extract', 'scheduled__2023-01-22T00:00:00+00:00', '--job-id', '3486', '--raw', '--subdir', 'DAGS_FOLDER/NSE_Open_Interest.py', '--cfg-path', '/tmp/tmpcb1pdn_g']
[2023-02-02T03:10:49.416+0530] {standard_task_runner.py:83} INFO - Job 3486: Subtask extract
[2023-02-02T03:10:49.482+0530] {task_command.py:389} INFO - Running <TaskInstance: NSE_Open_Interest.extract scheduled__2023-01-22T00:00:00+00:00 [running]> on host DESKTOP-SQIOQPD.
[2023-02-02T03:10:49.561+0530] {taskinstance.py:1511} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=nikheleshbhattacharyya@gmail.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=NSE_Open_Interest
AIRFLOW_CTX_TASK_ID=extract
AIRFLOW_CTX_EXECUTION_DATE=2023-01-22T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=3
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-22T00:00:00+00:00
[2023-02-02T03:10:49.562+0530] {logging_mixin.py:137} INFO - 2020-02-14 00:00:00
[2023-02-02T03:13:00.404+0530] {logging_mixin.py:137} INFO - 2020-02-15 00:00:00
[2023-02-02T03:13:03.574+0530] {logging_mixin.py:137} INFO - 2020-02-16 00:00:00
[2023-02-02T03:13:06.609+0530] {logging_mixin.py:137} INFO - 2020-02-17 00:00:00
[2023-02-02T03:13:09.642+0530] {logging_mixin.py:137} INFO - 2020-02-18 00:00:00
[2023-02-02T03:13:12.688+0530] {logging_mixin.py:137} INFO - 2020-02-19 00:00:00
[2023-02-02T03:13:15.724+0530] {logging_mixin.py:137} INFO - 2020-02-20 00:00:00
[2023-02-02T03:13:18.835+0530] {logging_mixin.py:137} INFO - 2020-02-21 00:00:00
[2023-02-02T03:13:21.927+0530] {logging_mixin.py:137} INFO - 2020-02-22 00:00:00
[2023-02-02T03:13:25.093+0530] {logging_mixin.py:137} INFO - 2020-02-23 00:00:00
[2023-02-02T03:13:28.129+0530] {logging_mixin.py:137} INFO - 2020-02-24 00:00:00
[2023-02-02T03:13:31.171+0530] {logging_mixin.py:137} INFO - 2020-02-25 00:00:00
[2023-02-02T03:13:34.217+0530] {logging_mixin.py:137} INFO - 2020-02-26 00:00:00
[2023-02-02T03:15:44.249+0530] {logging_mixin.py:137} INFO - 2020-02-27 00:00:00
[2023-02-02T03:17:57.573+0530] {taskinstance.py:1772} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/usr/local/lib/python3.8/dist-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/root/airflow/dags/NSE_Open_Interest.py", line 83, in extract
    dd,values = get_data(date)
TypeError: cannot unpack non-iterable int object
[2023-02-02T03:17:57.580+0530] {taskinstance.py:1322} INFO - Marking task as UP_FOR_RETRY. dag_id=NSE_Open_Interest, task_id=extract, execution_date=20230122T000000, start_date=20230201T214049, end_date=20230201T214757
[2023-02-02T03:17:57.592+0530] {standard_task_runner.py:100} ERROR - Failed to execute job 3486 for task extract (cannot unpack non-iterable int object; 4827)
[2023-02-02T03:17:57.616+0530] {local_task_job.py:159} INFO - Task exited with return code 1
[2023-02-02T03:17:57.633+0530] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
